src/styles/blocks/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$color-white: #fff;
+$color-main-accent: #060b35;
```

Comment:
Using descriptive color names in your SCSS variables might improve readability and maintainability.
Without understanding the design context, it might not be clear what `$color-primary` or `$color-secondary` refer to in terms of actual color.